zeth-contracts/contracts/Groth16Verifier.sol

```diff
@@ -0,0 +1,60 @@
+pragma solidity ^0.5.0;
+
+/*
+ * Reference papers:
+ *
+ * [BCTV14]:
```

Comment:
Useless reference of BCTV for the Groth16 snark

Reply:
Removed all groth16 references.